### ISSUE TITLE
After clicking on uncontrolled dropdown, clickin on a controlled dropdown will cause an exception

### ISSUE THREAD

**Issue by chanan** [chanan/BlazorStrap#100]
Can reproduce here: /BlazorStrap/navbars

Close the uncontrolled dropdown, then open the controlled one. Error in the log:

```
blazor.webassembly.js:1 WASM: System.InvalidOperationException: Id must be used with Drop Down Manager
blazor.webassembly.js:1 WASM:   at BlazorStrap.util.DropDownMenuHandler.IsOpen (System.String id) [0x0002e] in C:\Users\cbraunstein\source\repos\BlazorStrap\src\BlazorStrap\util\DropDownMenuHandler.cs:95 
blazor.webassembly.js:1 WASM:   at BlazorStrap.NavItem.LostFocus () [0x00017] in C:\Users\cbraunstein\source\repos\BlazorStrap\src\BlazorStrap\NavItem.razor:69 
```

**Comment by chanan** [comment]
Looks like onfocusout isnt firing at the right time, and when it is firing someone it can't find the `DropDownMenuControl.Id`

**Comment by chanan** [comment]
@jbomhold3 If you have time can you take a look at this?

**Comment by jbomhold3** [comment]
Fixed in https://github.com/chanan/BlazorStrap/pull/103

**Comment by chanan** [comment]
Thanks!

**Comment by jbomhold3** [comment]
No worries sorry about the delay was working on my library.

**Comment by chanan** [comment]
will be released in v1.0.0-preview6-20
